milestones/gain_awareness_of_ansible_meetups.md

```diff
@@ -0,0 +1,128 @@
+
+### Inquisitive
+
+The only stupid question is the one that does not get asked. We encourage our users to ask early and ask often. Rather than asking whether you can ask a question (the answer is always yes!), instead, simply ask your question. You are encouraged to provide as many specifics as possible. Code snippets in the form of Gists or other paste site links are almost always needed in order to get the most helpful answers. Refrain from pasting multiple lines of code directly into the chat channels - instead use gist.github.com or another paste site to provide code snippets.
+
+### Helpful
+
+The Ansible community is committed to being a welcoming environment for all users, regardless of skill level. We were all beginners once upon a time, and our community cannot grow without an environment where new users feel safe and comfortable asking questions. It can become frustrating to answer the same questions repeatedly; however, community members are expected to remain courteous and helpful to all users equally, regardless of skill or knowledge level. Avoid providing responses that prioritize snideness and snark over useful information. At the same time, everyone is expected to read the provided documentation thoroughly. We are happy to answer questions, provide strategic guidance, and suggest effective workflows, but we are not here to do your job for you.
+
+### ****Anti-harassment policy
+
+Harassment includes (but is not limited to) all of the following behaviors:
+
+- Offensive comments related to gender (including gender expression and identity), age, sexual orientation, disability, physical appearance, body size, race, and religion
+
+- Derogatory terminology including words commonly known to be slurs
+
+- Posting sexualized images or imagery in public spaces
+
+- Deliberate intimidation
+
+- Stalking
+
+- Posting others’ personal information without explicit permission
+
+- Sustained disruption of talks or other events
+
+- Inappropriate physical contact
+
+- Unwelcome sexual attention
+
+Participants asked to stop any harassing behavior are expected to comply immediately. Sponsors are also subject to the anti-harassment policy. In particular, sponsors should not use sexualized images, activities, or other material. Meetup organizing staff and other volunteer organizers should not use sexualized attire or otherwise create a sexualized environment at community events.
+
+In addition to the behaviors outlined above, continuing to behave a certain way after you have been asked to stop also constitutes harassment, even if that behavior is not specifically outlined in this policy. It is considerate and respectful to stop doing something after you have been asked to stop, and all community members are expected to comply with such requests immediately.
```

Comment:
```suggestion
In addition to the behaviors outlined above, continuing to behave in a certain way after you have been asked to stop also constitutes harassment, even if that behavior is not specifically outlined in this policy. It is considerate and respectful to stop doing something after you have been asked, and all community members are expected to comply with such requests immediately.
```

Reply:
I don't know if I really agree with this part. I mean, I agree with the spirit of what this text says. What I'm unsure of is if this is too vague. The second sentence goes a little overboard imo too.

Reply:
A separate PR to CoC.